[
    {
        "repo_name": "gitpython-developers/GitPython",
        "CVE_ID": "CVE-2023-40267",
        "Problem_Type": "NVD-CWE-noinfo",
        "Description": "GitPython before 3.1.32 does not block insecure non-multi options in clone and clone_from. NOTE: this issue exists because of an incomplete fix for CVE-2022-24439.",
        "URL": "https://github.com/gitpython-developers/GitPython/pull/1609",
        "Commit SHA": "ca965ecc81853bca7675261729143f54e5bf4cdd",
        "Tag": "['Issue Tracking', 'Patch']"
    },
    {
        "repo_name": "gitpython-developers/GitPython",
        "CVE_ID": "CVE-2024-22190",
        "Problem_Type": "CWE-426",
        "Description": "GitPython is a python library used to interact with Git repositories. There is an incomplete fix for CVE-2023-40590. On Windows, GitPython uses an untrusted search path if it uses a shell to run `git`, as well as when it runs `bash.exe` to interpret hooks. If either of those features are used on Windows, a malicious `git.exe` or `bash.exe` may be run from an untrusted repository. This issue has been patched in version 3.1.41.",
        "URL": "https://github.com/gitpython-developers/GitPython/pull/1792",
        "Commit SHA": "ef3192cc414f2fd9978908454f6fd95243784c7f",
        "Tag": "['Patch', 'Vendor Advisory']"
    }
]